rebasehelper/cli.py

```diff
@@ -143,6 +150,13 @@ def add_args(self):
             help="Specify number of retries in case build fails.",
             type=int
         )
+        self.parser.add_argument(
+            "--enable-option",
```

Comment:
--enable-extra-builder-options ? or --enable-builder-options ? Or you have any other suggestions?